src/core/qfieldcloud/qfieldcloudprojectsmodel.cpp

```diff
@@ -218,6 +220,28 @@ QFieldCloudProject *QFieldCloudProjectsModel::findProject( const QString &projec
   return nullptr;
 }
 
+void QFieldCloudProjectsModel::appendProject( const QString &projectId )
+{
+  if ( !mCloudConnection )
+    return;
```

Comment:
Lately I have seen a push we don't have `if` statements without `{}`. I guess it would be nice to have it in contributing guidelines which one do we prefer.